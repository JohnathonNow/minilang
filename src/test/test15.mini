def X := 100 + 200

def Y := X + 20

print('X = {X}\n')
print('Y = {Y}\n')

print('Test = {:([1, 2, 3, 4])}\n')
