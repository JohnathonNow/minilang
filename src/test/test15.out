X = 300
Y = 320
Test = 1 2 3 4
